### PR TITLE
DTSPO-13990 testing frontdoor migration

### PR DESCRIPTION
**Pull request by Tyler-35**
### JIRA link (if applicable) ###
[DTSPO-13990](https://tools.hmcts.net/jira/browse/DTSPO-13990)

### Change description ###
adding public dns for new frontend to test frontdoor migration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: environments/sandbox.yml [hunk 1]
```diff
@@ -172,6 +172,10 @@ cname:
     ttl: 300
     record: "sdshmcts-sbox.azurefd.net"
     shutter: false
+  - name: "toffee2"
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
```
Comment (timja) [inline environments/sandbox.yml:177]: this should be your new frontdoor instance not an existing one

Comment (Tyler-35) [reply environments/sandbox.yml]: updated with the new frontdoor

File: environments/sandbox.yml [hunk 2]
```diff
@@ -172,6 +172,10 @@ cname:
     ttl: 300
     record: "sdshmcts-sbox.azurefd.net"
     shutter: false
+  - name: "toffee2"
+    ttl: 300
+    record: "test-sbox.azurefd.net"
```
Comment (timja) [inline environments/sandbox.yml:177]: should have hmcts in the name as these are globally namespaced someone else could have this

Comment (Tyler-35) [reply environments/sandbox.yml]: updated it with hmcts-test-sbox.azurefd.net

**Review by timja — APPROVED** [review]

**Pull request merged**